plugins/arElasticSearchPlugin/lib/model/arElasticSearchModelBase.class.php

```diff
@@ -45,6 +45,11 @@ public function getCount()
     return $this->count;
   }
 
+  public function setCount($count)
```

Comment:
I don't see how this related to the $skip code either.

Reply:
This sets the "limit" of the indexing according to the logic.